embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/internal/logs/EmbraceLogRecordData.kt

```diff
@@ -0,0 +1,43 @@
+package io.embrace.android.embracesdk.internal.logs
+
+import com.squareup.moshi.Json
+import com.squareup.moshi.JsonClass
+import io.opentelemetry.sdk.logs.data.LogRecordData
+
+/**
+ * Serializable representation of [EmbraceLogRecordData]
+ */
+@JsonClass(generateAdapter = true)
+internal data class EmbraceLogRecordData(
+    @Json(name = "trace_id")
+    val traceId: String,
+
+    @Json(name = "span_id")
+    val spanId: String,
+
+    @Json(name = "time_unix_nano")
+    val timeUnixNanos: Long,
+
+    @Json(name = "severity_number")
+    val severityNumber: Int,
+
+    @Json(name = "severity_text")
+    val severityText: String?,
+
+    @Json(name = "body")
+    val body: EmbraceBody,
+
+    @Json(name = "attributes")
+    val attributes: Map<String, String> = emptyMap()
+) {
+
+    internal constructor(logRecordData: LogRecordData) : this(
```

Comment:
nit: would be good to add a test case that converts this type to `EmbraceLogRecordData`

Reply:
Added a test case for the constructor.